docs_espressif/en/additionalfeatures/coverage.rst

```diff
@@ -0,0 +1,70 @@
+
+Click menu **View** -> **Command Palette...** and type **ESP-IDF: Show Examples Projects** and choose ``Use Current ESP-IDF (/path/to/esp-idf)``. If the user doesn't see the option, please review the setup in Installation <installation>.
+
+A window will be open with a list a projects, go the **system** section and choose the `gcov`. You will see a **Create Project Using Example gcov** button in the top and a description of the project below. Click **Create Project Using Example GCOV** button.
+
+.. image:: ../../../media/tutorials/coverage/gcov_example.png
+
+Now select a container directory where to copy the example project. For example, if the user choose ``/Users/myUser/someFolder`` the resulting folder will be ``/Users/myUser/someFolder/gcov``. This new project directory will be created and opened in Visual Studio Code.
+
+First the user should select an Espressif target (esp32, esp32s2, etc.) with the **ESP-IDF: Set Espressif Device Target** command. Default is ``esp32`` and the one used in this tutorial.
+
+Next configure your sdkconfig project with the **ESP-IDF: Configure Project SDKConfig for Coverage** command or by yourself using the **ESP-IDF: SDK Configuration Editor** command where the user can modify the ESP-IDF project settings. After all changes are made, click save and close this window.
+
+.. image:: ../../../media/tutorials/basic_use/gui_menuconfig.png
+
+The example will enable the following options by default:
+
+- Enable the Application Tracing Module under ``Component Config`` -> ``Application Level Tracing`` -> ``Data Destination`` by choosing `Trace Memory`.
+- Enable GCOV to host interface under ``Component Config`` ->`` Application Level Tracing`` -> ``GCOV to Host Enable``.
+- Enable OpenOCD Debug Stubs under ``Component Config`` -> ``ESP32-specific`` -> ``OpenOCD Debug Stubs``
+
+Now to build the project, flash your device and start the ESP-IDF Monitor you can use the **ESP-IDF: Build your Project**, **ESP-IDF: Flash your Project** and **ESP-IDF: Monitor Device** commands.
+
+.. note::
+  There is also a **ESP-IDF: Build, Flash and Start a Monitor on your Device** command which does all three former commands in one.
+
+Next step is to launch OpenOCD and send some commands. To start openOCD from the extension, execute the **ESP-IDF: OpenOCD Manager** command or from the ``OpenOCD Server (Running | Stopped)`` button in the Visual Studio Code status bar. OpenOCD server output is shown in menu **View** -> **Output** -> **ESP-IDF**.
+
+Launch a new terminal with menu Terminal -> New Terminal and execute ``telnet <oocd_host> <oocd_port>`` which by default is ``telnet localhost 4444``. Latest macOS users can use ``nc <oocd_host> <oocd_port>`` if ``telnet`` is not in the system.
+
+.. note::
+  The user can modify **openocd.tcl.host** and **openocd.tcl.port** configuration settings to modify these values.
```

Comment:
For consistency in addressing the reader, I would suggest addressing him directly and not use the `user` word here